src/QMCTools/ppconvert/src/common/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+add_library(common EXCLUDE_FROM_ALL "")
+
+target_sources(common
+  PRIVATE
+  "${CMAKE_CURRENT_SOURCE_DIR}/GKIntegration.cc"
```

Comment:
Similar to src/CMakeLists.txt, the ${CMAKE_CURRENT_SOURCE_DIR} should not be needed.

Reply:
Specifying this is just part of the idiom as I learned it, perhaps because if the sources are moved to another directory and brought in via an include() it is changed to ${CMAKE_CURRENT_LIST_DIR} and this emphasizes these sources are not in a separate location.

Reply:
As well as making regular expression update trivial.

Reply:
Okay. Looking this up I see the add_library call could be in a different directory from the target_sources call, and the directories searched are relative to the location of the add_library call - adding the CMAKE_CURRENT_SOURCE_DIR allows them to still be found.